File: Official Windows Platform Sample/XAML user and custom controls sample/[C#]-XAML user and custom controls sample/C#/Shared/BasicCustomControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Windows.UI.Xaml;
using Windows.UI.Xaml.Controls;
using Windows.UI.Xaml.Data;
using Windows.UI.Xaml.Documents;
using Windows.UI.Xaml.Input;
using Windows.UI.Xaml.Media;

namespace UserAndCustomControls
{
    public sealed class BasicCustomControl : Control
    {
        public BasicCustomControl()
        {
            this.DefaultStyleKey = typeof(BasicCustomControl);
        }
    }
}

File: Official Windows Platform Sample/Windows 8 app samples/[VB]-Windows 8 app samples/VB/Windows 8 app samples/XAML user and custom controls sample (Windows 8)/VB/BasicCustomControl.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports Windows.UI.Xaml
Imports Windows.UI.Xaml.Controls
Imports Windows.UI.Xaml.Data
Imports Windows.UI.Xaml.Documents
Imports Windows.UI.Xaml.Input
Imports Windows.UI.Xaml.Media

Public NotInheritable Class BasicCustomControl
    Inherits Control
    Public Sub New()
        Me.DefaultStyleKey = GetType(BasicCustomControl)
    End Sub
End Class
